### PR TITLE
309294-Configure-embedConfigjson-instead-of-EmbedProperties-in-BlazorWebAssembly-Sample

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: BlazorWebAssembly/Shared/WeatherForecast.cs [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
-        public int TemperatureC { get; set; }
-
-        public string Summary { get; set; }
-
-        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
-    }
-}
+namespace BlazorWebAssembly.Shared;
+
+public class WeatherForecast
```
Comment (vinothkumar1732) [inline BlazorWebAssembly/Shared/WeatherForecast.cs]: Why you weren't properly cleanup the sample?
image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,67 @@
...
+
+    ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+  * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+  
+    ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+    ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+  * Copy the downloaded `embedConfig.json` file and paste it into the designated [location](https://github.com/boldbi/blazor-webassembly-sample/tree/master/BlazorWebAssembly/Server) within the application. Please ensure you have placed it in the application, as shown in the following image.
```
Comment (vinothkumar1732) [inline README.md]: Here we are moving the config.json into Server folder. So you should mark the server folder in the image too. And also mention the Server folder in text also. You did simply linked the Server folder alone. That is not enough.

File: README.md [hunk 3]
```diff
@@ -1,70 +1,67 @@
...
+
+    ![EmbedConfig image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/5e3e876f-ceff-435f-835c-99e51f709c2d)
+
+## How to run sample using command prompt
+
+  1. Open the command line interface and navigate to the specified file [location](https://github.com/boldbi/blazor-webassembly-sample/tree/master/BlazorWebAssembly) where the project is located.
+
+  2. Open the terminal and navigate to the `Server project` directory using the cd command(e.g., cd C:\BlazorProject\Server).
```
Comment (vinothkumar1732) [inline README.md]: Highlight the command always.

File: README.md [hunk 4]
```diff
@@ -1,70 +1,67 @@
...
+  
+  3. Finally, run the application using the command `dotnet watch run`. After executing the command, the application will automatically launch in the default browser. You can access it at the specified port number (e.g., http://localhost:5154/).
```
Comment (vinothkumar1732) [inline README.md:42]: Would application automatically launch in browser?

File: README.md [hunk 5]
```diff
@@ -1,70 +1,67 @@
...
-    </tr>
-  </tbody>
-</table>
-
-Please refer to the [help documentation](https://help.boldbi.com/embedded-bi/javascript-based/samples/v3.3.40-or-later/blazor-with-javascript/#how-to-run-the-blazor-webassembly-sample) to learn how to run the sample.
-
-## Online Demos
+  1. Open the [Blazor WebAssembly](https://github.com/boldbi/blazor-webassembly-sample/tree/master/BlazorWebAssembly) sample in Visual Studio Code.
+
+  2. Open the terminal and navigate to the `Server project` directory using the cd command(e.g., cd C:\BlazorProject\Server).
```
Comment (vinothkumar1732) [inline README.md]: Highlight the command.

File: README.md [hunk 6]
```diff
@@ -1,70 +1,67 @@
...
+   
+  3. To run the application, use the command `dotnet watch run` in the terminal. After executing the command, the application will automatically launch in the default browser. You can access it at the specified port number (e.g., http://localhost:5154/).
```
Comment (vinothkumar1732) [inline README.md:54]: Would application automatically launch in browser?

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]

File: BlazorWebAssembly/Shared/WeatherForecast.cs (hunk 1 above)
Comment (ghost) [reply BlazorWebAssembly/Shared/WeatherForecast.cs]: Unwanted files all are removed.

File: README.md (hunk 4 above)
Comment (ghost) [reply README.md:42]: YEs

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: Command highlighted

File: README.md (hunk 6 above)
Comment (ghost) [reply README.md:54]: yes

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: Command line updated

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Content and image updated.

**Review by vinothkumar1732 — APPROVED** [review]
Sample and readme file changes are fine.

**Pull request merged**